File: trip_parser/v2/get/trip_parser_jobs_jobId/Python SDK/trip_parser_status.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

# Retrieves the status of the process by passing the jobId returned by Trip Parser Jobs
try:
    response = amadeus.travel.trip_parser_jobs.status('JKKSJNMSKHJ').get()
    print(response.data['status'])
except ResponseError as error:
    raise error

File: trip_parser/v2/get/trip_parser_jobs_jobId/iOS SDK/TripParserStatus.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

amadeus.travel.tripParserJobs.status(jobId: "JKKSJNMSKHJ").get(
    onCompletion: {
        response, error in
        if error == nil {
            print(response!.data)
        }
                                        })
